securedrop/source_templates/lookup.html

```diff
@@ -17,11 +17,11 @@
   {% endif %}
 </div>
 
-<h1 class="headline">{{ gettext('Submit Materials') }}</h1>
+<h1 class="headline">{{ gettext('Submit Materials or Messages') }}</h1>
```

Comment:
:+1: to the generic term "files" - I've had several people be confused why we only accept "documents" assuming e.g. Microsoft Word documents

Reply:
I also like "files" better.